src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -131,6 +132,11 @@ def get_reduce_identity_value(op_name_: str, type_: Any):
 }
 
 
+# Define type of index variables and symbols used in definitions for array shape/strides
+_INDEX_DTYPE = _TYPE_MAPPING["int"]
+_SYMBOL_DTYPE = _TYPE_MAPPING["int64"]
+
+
 @dataclasses.dataclass
 class SymbolExpr:
     value: str | dace.symbolic.sympy.Basic
```

Comment:
Out of curiosity shouldn't it also be `dace.symbolic.SymExpr`?

Reply:
Either `dace.symbolic.SymExpr` or the union type `dace.symbolic.SymbolicType`, but probably better `dace.symbolic.SymExpr`, as you suggest, because it wraps `sympy`.

Reply:
I agree, I will change it to `dace.symbolic.SymbolicType`